Problem with setup.py

https://github.com/lsst-dm/alert-stream-simulator/blob/3a591ac857d2db931733e2b947de7468146a46d7/setup.py#L3-L11

When I tried to install the alert-stream-simulator with `make install` I got:

```bash
Processing dependencies for rubin-alert-stream-simulator==0.1.0
error: fastavro 1.1.0 is installed but fastavro<0.24 is required by {'lsst-alert-packet'}
make: *** [Makefile:5: install] Error 1
```

I specified `fastavro<0.24` in setup.py and all works!

`Finished processing dependencies for rubin-alert-stream-simulator==0.1.0`

--- a/setup.py
+++ b/setup.py
@@ -4,7 +4,7 @@ install_requires = [
     'avro-python3',
     'confluent_kafka',
     'astropy',
-    'fastavro',
+    'fastavro<0.24',
     'python-snappy',
     'lsst-alert-packet>=0.2',
     'lsst-alert-stream>=0.2',

